server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/Db2SqlParser.g4

```diff
@@ -0,0 +1,1698 @@
+/*
```

Comment:
A lot of rules produce "contains an optional block with at least one alternative that can match an empty string" warning